homeassistant/components/smartthings/sensor.py

```diff
@@ -403,6 +411,13 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
                         for index in range(len(THREE_AXIS_NAMES))
                     ]
                 )
+            elif capability == Capability.power_consumption_report:
+                sensors.extend(
+                    [
+                        SmartThingsPowerConsumptionSensor(device, index)
+                        for index in range(len(POWER_CONSUMPTION_REPORT_NAMES))
```

Comment:
```suggestion
                        SmartThingsPowerConsumptionSensor(device, report_name)
                        for report_name in POWER_CONSUMPTION_REPORT_NAMES
```

than use `report_name`  in `SmartThingsPowerConsumptionSensor` instead of always gather index XYZ from `POWER_CONSUMPTION_REPORT_NAMES`